INSTALL.md

```diff
@@ -194,7 +188,96 @@ Run [NPM](https://www.npmjs.com/) to install all the required Node modules:
 npm install
 ```
 
-#### Setup Database
+## Manual setup on Fedora (29)
+
+### Prerequisites
+
+#### Ruby version
+
+The project supports **Ruby 2.3.x**.
```

Comment:
I think that from now on we are only supposed to support at least `2.4.X`, but I am not sure. @hallelujah knows better.

Reply:
👍

Reply:
@Martouta  @hallelujah,  Is it a good idea to have different ruby versions  on CI (>= 2.3) so we can exactly now which versions do we support?

Reply:
changed it to 2.4

Reply:
Only 2.4 :)
2.3 is EOL and we removed support of it
2.5 is blocked by rails upgrade and many gems update